src/Pollers/DeviceMappingPoller.php

```diff
@@ -222,6 +231,8 @@ private function determineDeviceTypes(array $chunks)
         $updatedChunks = [];
         foreach ($chunks as $host)
         {
+			///TODO Exit out for ICMP only devices, do not waste resources on them.
+			//if($templateDetails['snmp_community'] == "disabled") continue;
```

Comment:
Can you strip out the TODOs for the PR and then submit these changes later if you plan to?

Reply:
todos removed or implemented